packages/xcoobee-cookie-kit-react/src/CookieKitPopup.js

```diff
@@ -1,16 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
 import ReactCountryFlag from "react-country-flag";
-// import Enums from "xcoobee-enums";
 
 import {
   cookieDefns as allAvailCookieDefns,
   cookieTypes,
   locales,
   links,
 } from "xcoobee-cookie-kit-core/src/configs";
-import renderText from "xcoobee-cookie-kit-core/src/renderText";
 import countryData from "xcoobee-cookie-kit-core/src/countryData";
```

Comment:
Correct me if I am wrong but I think this import should be
```suggestion
import { countryCodes } from "xcoobee-cookie-kit-core/src/countryData";
```
since we are not doing a default export.
The variable below will need to be updated too.